pkg/network/vmispec/vmispec_suite_test.go

```diff
@@ -1,3 +1,22 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
```

Comment:
what's the point in this ?

Reply:
this file was simply missing  license signature, so I added it.

Reply:
but you don't touch anything else in the file, it's surprising to see this change here.

(not saying to drop it).

Reply:
yeah it's a little piggy back I added, If you'll insist I'll add it in a separate PR.

Reply:
Feel free to keep it.